src/Symfony/Component/Debug/Debug.php

```diff
@@ -53,6 +53,7 @@ public static function enable($errorReportingLevel = null, $displayErrors = true
             ini_set('display_errors', 1);
         }
         $handler = ErrorHandler::register();
+        $handler->setDefaultLogger(new BootstrappingLogger());
```

Comment:
there should be a way to disable it IMO, to avoid filling memory in case you never set a real logger later